testing-framework.js

```diff
@@ -0,0 +1,10 @@
+const assertEquals = (expectedOutput, actualOutput) => expectedOutput === actualOutput;
+const assertLessThan = (expectedOutput, actualOutput) => expectedOutput < actualOutput;
+const assertTrue = actualOutput => actualOutput === true;
+const assertFalse = actualOutput => actualOutput === false;
+const assertNotEquals = (expectedOutput, actualOutput) => expectedOutput !== actualOutput;
+
```

Comment:
I should've implemented other types of testing as I only used the assertEquals for all my testing purposes.